src/collectd.conf.in

```diff
@@ -665,11 +665,11 @@
 #	MountPoint "/home"
 #	FSType "ext3"
 #	IgnoreSelected false
+#
 #	LogOnce false
-#	ReportByDevice false
 #	ReportInodes false
-#	ValuesAbsolute true
-#	ValuesPercentage false
+#	ReportUsage true
+#	ReportUtilization false
```

Comment:
While I haven't complained about these in other PRs, I think these names are too similar i.e. easily confused.  Adding unit to their names would make them more understandable without user needing to resort to documentation:
```suggestion
#	UsageBytes true
#	UtilizationRatio false
```

What do you think?